content/docs/command-reference/params/index.md

```diff
@@ -143,6 +143,61 @@ $ dvc run -n train -d logs/ -o users.csv \
           python train.py
 ```
 
+## Examples: Python parameters file
+
+Consider this parameters file in Python format, named `params.py`:
+
+```python
+IS_BOOL: bool = True
+CONST = 5
+
+
+class TrainConfig:
+    EPOCHS = 70
+
+    def __init__(self):
+        self.layers = 9
+
+
+class TestConfig:
+    TEST_DIR = "path"
+    METRICS = ["metric"]
+```
+
+The following [stage](/doc/command-reference/run) depends on params
+`IS_BOOL`, `CONST`, as well as `TrainConfig`'s `EPOCHS` and `layers`:
```

Comment:
@aandrusenko qq - if we depend on the instance variable (`self.layers`), when do we extract value? What is the semantics here?

Reply:
From a class, you can extract either class constants or `self` variables defined in` __init__`

Reply:
Yep, thanks! But what about the semantics? What if I have

```
self.layers = 10
self.layers = 12
```

or if I have

```
self.layers = 3+2
```

what are the expectations and limitations here?

Reply:
In the first case, `TrainConfig.layers` will be `12`. In the second, `TrainConfig.layers` will not be found because the complex expression

Reply:
let's make it part of the docs and/or example? @aandrusenko it's totally fine if you don't have enough capacity, let us know and we'll take it over, but I think we should specify some basic semantics for this. Modifying example (add a few lines with Python comments + some text ?) should be enough, I guess.

Reply:
@shcheklein, I added more comments